tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>I think that the flow is more:</p><p> </p><ol><li>Create TrotterEvolutionProblem</li><li>Create Trotter QRTE instance</li></ol><p> </p><p>So maybe you can just swap these sections.</p><p> </p><p>It would also be nice to see an introductory sentence to the TrotterQRTE class, you mention later that there are several features, or tasks it can be used for. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>